maquetado/setup/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training">
+    <meta name="author" content="Wolox">
+    <link rel="stylesheet"  type="text/css" href="application.css">    
+  </head>
+  <body>    
+      <h2 class="title">Front-End Tools</h2>      
```

Comment:
You have more than two spaces here, check you IDE configuration, maybe it is changing your code